.travis.yml

```diff
@@ -47,5 +56,10 @@ script:
       else
         mvn org.jacoco:jacoco-maven-plugin:prepare-agent install org.jacoco:jacoco-maven-plugin:report sonar:sonar;
       fi
+
+    - export http_proxy=""
+    - export https_proxy=""
+    - ant dockerimage
```

Comment:
looks like we are building OSC twice. 

We should just be able to build OSC once(with `mvn org.jacoco:jacoco-maven-plugin:prepare-agent install org.jacoco:jacoco-maven-plugin:report sonar:sonar`) and add the output binary to the docker image.